lib/search.js

```diff
@@ -86,14 +87,30 @@ module.exports = (elastic, queryParams, next) => {
   const searchOpts = {
     index: 'smg',
     from: pageNumber * pageSize,
-    size: pageSize,
     body: body
   };
 
   if (!queryParams.q) {
     searchOpts.body.query.function_score.query = {match_all: {}};
   }
 
+  if (queryParams.random) {
```

Comment:
a comment linking to the issue/story where this functionality was requested would be good here `else` the "_next person_" will have to _waste time_ asking: "_**Why** is this code here...?_"